### DISCUSSION TITLE
Inside Peach's Castle 1F

### DISCUSSION THREAD

**Discussion by JCR64** [nzp-team/nzportable#936] [Map Releases]
_Another_ N64 remake, this time Inside Peach's Castle from Super Mario 64, just the 1F. Much smaller scale than the other remakes so hopefully it plays nicer on non-FTE platforms.

Also included is a retextured version using textures from Knave texture pack for Quake that might suit PSP better. It is a slightly tall version of the level and has no coins/music.

Easter Eggs:

- Coins give points
- Music (Hint: Tower of the Wing Cap)

Credits:

- Nintendo
- Knave

# [DOWNLOAD](https://github.com/user-attachments/files/17333741/sm64_insidecastle1f.zip)

![screen1](https://github.com/user-attachments/assets/e7ed6074-fa3f-4182-ba73-35cd825fc195)

![screen2](https://github.com/user-attachments/assets/e2236c73-5323-465d-9223-df9622e2736f)

# [DOWNLOAD (Knave Edition)](https://github.com/user-attachments/files/17333744/sm64_insidecastle1f_knave.zip)

![screen3](https://github.com/user-attachments/assets/aa9e802c-a26b-453e-8fdf-55d0aa17f114)

# [Video Playthrough](https://www.youtube.com/watch?v=6ojZVJZs7js)

## Known issues:

- The rotating coins cause a crash on PSP so please replace the .bsp file with this one for stationary coins. [DOWNLOAD](https://github.com/user-attachments/files/17333756/sm64_insidecastle1f_norotation.zip)

**Comment by LegendDD44** [comment]
![Screenshot 2024-10-11 150158](https://github.com/user-attachments/assets/bbbc596f-c6f9-4f4f-9c38-52e4a9219cce)
my first try

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard since you have different download links for different platforms, and the downloads are not hosted on archive.org

**Comment by Ellis243454** [comment]
This works great on old 3ds both versions fun map!

**Comment by mmkkiinn** [comment]
The map has pap

**Comment by sharkgamersocool-cloud** [comment]
how do i add the sound i put it in sound folder but doesnt work

**Comment by tysonbulls64-hash** [comment]
pls make floor 2 or the whole castle
